migtests/scripts/resumption.py

```diff
@@ -155,30 +157,54 @@ def prepare_import_data_command(config):
 
     return args
 
+def inject_varying_flags(command):
+    global varying_flags
+
+    for flag, setting in varying_flags.items():
+        value_list = setting["value"]
+        if setting["type"] == "range":
+            value = random.randint(value_list[0], value_list[1])
+        elif setting["type"] == "choice":
+            value = random.choice(value_list)
```

Comment:
what does choice means?
whats the d/f b/w choice() and randint() here

Reply:
"Range" means a random integer will be picked from a start to end range using random.randint().
"Choice" means the value will be randomly picked from a predefined list of options in the config file using random.choice().

Sample config for clarity:
```
--parallel-jobs:
    type: range
    value: [1, 10]
  --batch-size:
    type: choice
    value: [1234, 2500, 10500, 20000]
```